chimney/src/main/scala/io/scalaland/chimney/internal/DerivationError.scala

```diff
@@ -67,8 +67,22 @@ object DerivationError {
               s"  derivation from $fieldName: $sourceTypeName to $targetTypeName is not supported in Chimney!"
           }
 
+          val fieldsWithMethodAccessor = errors.collect {
+            case MissingAccessor(fieldName, _, _, _, true) => s"`$fieldName`"
+          }
+          val methodAccessorHint =
+            if (fieldsWithMethodAccessor.nonEmpty) {
+              val first3Fields = fieldsWithMethodAccessor.take(3).mkString(", ")
+              val otherFields = fieldsWithMethodAccessor.length - 3
+              val fields = if (otherFields > 0) s"$first3Fields and $otherFields other methods" else first3Fields
+
+              s"There are methods in $sourceTypeName that might be used as accessors for $fields fields in $targetTypeName. Consider using `.enableMethodAccessors`"
+            } else ""
+
           s"""$targetTypeName
            |${errStrings.mkString("\n")}
+           |
```

Comment:
perhaps additional output line when there's no hint could be avoided?

Reply:
👌